source/_integrations/shell_command.markdown

```diff
@@ -44,7 +44,7 @@ Any service data passed into the service call to activate the shell command will
 The `command` is executed within the [configuration directory](/docs/configuration/).
 
 <div class='note'>
-If you are using [Home Assistant operating system](https://github.com/home-assistant/operating-system), the commands are executed in the `homeassistant` container context. So if you test or debug your script, it might make sense to do this in the context of this container to get the same runtime environment.
+If you are using <a href="https://github.com/home-assistant/operating-system">Home Assistant operating system</a>, the commands are executed in the <code>homeassistant</code> container context. So if you test or debug your script, it might make sense to do this in the context of this container to get the same runtime environment.
 </div>
```

Comment:
This is not the correct fix. Please revert the change and add a new line after the opening div and before the closing div instead.

The new line will allow the parser to switch context between Markdown and HTML.